crossref/restful.py

```diff
@@ -134,14 +136,18 @@ class Endpoint:
 
     def __init__(
         self,
+        backend=None,
```

Comment:
backend parameter should be the last one. Some users don't use named parameters and your commit will break their scripts.

Reply:
I thought, since all the parameters had default values, if I gave backend also a default value it wouldn't break anything. But it is something that I can change it, so OK.

Reply:
I gave a second thought, and I understood what you are saying. You're right. I will change it as soon as possible.